kratos/utilities/particles_utilities.h

```diff
@@ -0,0 +1,330 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Riccardo Rossi
+//
+
+#pragma once
+
+// System includes
+#include <string>
+#include <iostream>
+
+
+// External includes
+
+
+// Project includes
+#include "includes/define.h"
+#include "utilities/binbased_fast_point_locator.h"
+#include "utilities/atomic_utilities.h"
+
+namespace Kratos
+{
+///@addtogroup ApplicationNameApplication
+///@{
+
+///@name Kratos Globals
+///@{
+
+///@}
+///@name Type Definitions
+///@{
+
+///@}
+///@name  Enum's
+///@{
+
+///@}
+///@name  Functions
+///@{
+
+///@}
+///@name Kratos Classes
+///@{
+
+/// Collection of utilities to compute statistic of particles
+/** Detail class definition.
+*/
+class ParticlesUtilities
```

Comment:
I'd call this `ParticleTrackingUtilities` of something of this sort to avoid developers being confused with any PFEM-related stuff.

Reply:
ParticleTrackingUtilities remids me of pfem2...

Reply:
I also do not really love this renaming...honestly i would keep the name as it is

Reply:
Fine for me then.